extend-admin-ui-quick-start/README.md

```diff
@@ -0,0 +1,57 @@
+# Customizing Keycloak Themes
+
+The Keycloak Admin UI, which is the interface for managing Keycloak, is built with [PatternFly](https://www.patternfly.org/v4/) - a set of UI design patterns and resources that can be used to create consistent, user-friendly interfaces for web applications.
+
+## Creating Themes
+
+With themes, it is possible to configure the Admin UI's look and feel. To ensure consistency and maintainability of the custom Keycloak theme, we advise using PatternFly best practices when customizing the Keycloak Admin UI. This includes following the recommended folder structure and using PatternFly CSS classes and guidelines when creating custom styles and overrides.
+
+### Required Folder Structure
+
+The recommended folder structure for a custom Keycloak theme is as follows:
+
+├── mytheme/
+│ ├── admin/
+│ │ ├── resources/
+│ │ | ├── assets/
+│ │ | ├── css/
+│ │ | | ├── styles.css
+│ │ | ├── resources/
+│ │ ├── theme.properties
+
+In order to add a custom theme, in the `js/apps/keycloak-server/server/themes` folder, create a new folder named `mytheme` (can be any name you desire) which houses your personalized Admin UI theme. Within the `admin` sub-folder of this `mytheme` folder, there is a `theme.properties` file, along with a `resources` sub-folder that encompasses directories for CSS files, JavaScript files, and images. Within the `css` sub-folder, there is a `styles.css` file, which allows the addition of customized PatternFly styles. The `theme.properties` is a configuration file defines attributes such as the parent theme, and theme css. The `theme.properties` file is used to register the theme with the Keycloak server and integrate it with the Keycloak user interface.
```

Comment:
maybe we don't want to use `js/apps/keycloak-server/server/themes` here but instead just tell people to put it into `themes` folder of the keycloak server

Reply:
> maybe we don't want to use `js/apps/keycloak-server/server/themes` here but instead just tell people to put it into `themes` folder of the keycloak server
+1  We definitely do not want to pollute `js/apps/keycloak-server/server/themes`.

A couple of things to keep in mind here:
1. For final deployment, you should have the option to bundle as a jar.  We don't need to have that ability in this PR, but the quickstart should be able to build the theme as a jar.  See https://www.keycloak.org/docs/latest/server_development/#deploying-themes
2. In the WildFly version of Keycloak, it was possible to have more than one /themes directory.  So it would be possible to tell your server to point to your development directory when creating a theme.  I'm not sure if the Quarkus version has this, but if it doesn't we can add it if needed.

Reply:
Thanks @ssilvert I will create the next PR with a suggested solution of adding an option to download a Keycloak server and provide a way to start it with the --spi-theme-folder-dir=<path> and using Vite.